src/components/joinus/JoinUsHero.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import HeroPlaceholder from "@/assets/joinus/hero_placeholder.jpg";
```

Comment:
really small nitpick, but please use camel case
```suggestion
import heroPlaceholder from "@/assets/joinus/hero_placeholder.jpg";
```